process/sp.py

```diff
@@ -23,6 +23,9 @@
 import json
 import fiona
 import sys
+# something about the CRS definition (see config file) elicits a future warning for Proj6; ignore for now
+import warnings
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
CRS objects and definitions have changed in recent months. We should make sure these are up to date. Can you turn off the warning filtering? Better to see it.

Reply:
Done!